Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -272,7 +272,7 @@ public async Task PutObjectAsync(string bucketName, string objectName, Stream da
 
                 }
             }
-            
+
```

Comment:
```suggestion
```

Reply:
Implemented in all the files (in this patchset only) in which this was present.